<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventPage</class>
 <widget class="QWidget" name="EventPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color:rgb(51,153,255);
border:1px solid black;
font:12px &quot;SimHei&quot;;
color:white;
}

QPushButton
{
outline:none;
border-image: url(:/image/buttonUP.jpg);
font:20px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;
}
</string>
  </property>
  <widget class="QPushButton" name="btnCurrentPage">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>10</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>当前故障</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHistoryPage">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>60</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>历史故障</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLevel12">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>110</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>1／2级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLevel3">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>160</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>3级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTotal">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>210</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>全部</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPre">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>260</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>◀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>260</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>▶</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPage">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>300</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
font:15px;</string>
   </property>
   <property name="text">
    <string>2/100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDetail">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>330</y>
     <width>771</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid white;
border-radius:10px;
font:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border:0px;
font:25px;</string>
   </property>
   <property name="text">
    <string>操作提示</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
